src/modules/exam/exam.service.ts

```diff
@@ -0,0 +1,66 @@
+import { BadRequestException, ConflictException, Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { DeleteResult, Repository } from 'typeorm';
+import { UUID } from 'crypto';
+import { Exam } from 'src/entities/exam.entity';
+import { CreateExamDto } from './dto/create-exam.dto';
+import { UpdateExamDto } from './dto/update-exam.dto';
+
+@Injectable()
+export class ExamService {
+  constructor(
+    @InjectRepository(Exam)
+    private examRepository: Repository<Exam>
+  ) {}
+
+  async create(createExamDto: CreateExamDto): Promise<Exam> {
+    try {

```

Comment:
why only create have try catch, If there was exception interceptor at global, pls remove this try catch

Reply:
ok